pom.xml

```diff
@@ -40,47 +40,78 @@
             <artifactId>docker-java</artifactId>
             <version>3.2.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.java</groupId>
+            <artifactId>icing</artifactId>
+            <version>2020.09.21</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/src/main/resources/icing-2020.09.21.jar</systemPath>
+        </dependency>
         <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-api -->
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
             <version>2.13.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-resources-plugin</artifactId>
+            <version>3.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-params</artifactId>
-            <version>5.5.1</version>
+            <version>5.7.0</version>
```

Comment:
Btw why are we going back in versions?

Reply:
What do you mean? It's been changed to 5.7.0 from 5.5.1